docker-compose.circleci.yml

```
@@ -40,7 +40,7 @@ services:
       dockerfile: Dockerfile.client
       args:
         - REACT_APP_APP_ENV=$APP_ENV
-        - REACT_APP_API_ADDRESS=http://easi:8080/api/v1
+        - REACT_APP_API_ADDRESS=http://localhost:8080/api/v1
```

```suggestion
        - REACT_APP_API_ADDRESS=http://easi:8080/api/v1
```
I'm hopeful the comment above about adding `FLAG_SOURCE` will address the underlying issue. So I think this will need to remain as http://easi:8080/api/v1.